<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>381</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineLogin"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineMdp"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mot de passe</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="Annulation">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="Connexion">
         <property name="text">
          <string>Se connecter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>150</y>
        <width>501</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Precedent">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>300</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Précédent</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Annulation</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Connexion</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>seConnecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stackedWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineMdp</sender>
   <signal>returnPressed()</signal>
   <receiver>Connexion</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>547</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>seConnecter()</slot>
 </slots>
</ui>
